protocol/src/main/java/org/fido/iot/protocol/Const.java

```diff
@@ -189,12 +189,10 @@
   //EPID SigInfo Types
   public static final int SG_EPIDv10 = 90; // Intel EPID, version 1.0
   public static final int SG_EPIDv11 = 91; // Intel EPID, version 1.1
-  public static final int SG_EPIDv20 = 92; // Intel EPID, version 2.0
```

Comment:
Could you please help me understand why EPID20 constants are removed? Will we be supporting only EPID11 in this implementation?

Reply:
Yes, that is exactly correct - we no longer support EPID 2.0.

Reply:
ok